### DISCUSSION TITLE
Recommended way to extract out schema of a prop

### DISCUSSION THREAD

**Discussion by HugeLetters** [open-circle/valibot#681] [Q&A] [answered]
If I have an object schema is doing `schema.entries.prop` the recommended way to get a schema for a particular prop? 
I'm thinking what's that best way to define a schema for a form with Modular Forms and since the form is pretty big I need fine-grained validations. 
So I was thinking of defining a whole schema and then just applying individual schemas to fields

**Comment by HugeLetters** [comment]
Also a related question :eyes: 
What would be way to go to remap an object schema?
For form validation it should be just a basic schema - but I also need to store that data over the network and so when I get it back I need to have it parsed to make sure it has the appropriate shape but with a caveat: any invalid fields just get reset to a fallback value.

So I would need to remap the basic schema so that each field not has a fallback schema. 
```ts
const entries = Object.entries(schema.entries.prop);
const remappedEntries = entries.map(([key, schema])=>[key, v.fallback(schema, getFallback(key))]);
const fallbackSchema = Object.fromEntries(remappedEntries);
```
is this the way?

**Reply by fabian-hiller** [reply]
Yes, this can work, but sometimes it is easier to just define two schemas.

**Comment by fabian-hiller** [comment, marked as answer]
Yes, `schema.entries.prop` is the recommended way.
